packages/react/src/hooks/use-aria-conditional-ids.ts

```diff
@@ -0,0 +1,11 @@
+interface ConditionalValue {
+    id: string | undefined;
+    condition?: boolean;
```

Comment:
Pour que ce soit peut-être un peu plus cohérent avec le nom de l'interface?
```suggestion
    include?: boolean;
    value: string | undefined;
```